### PR TITLE
en: update get-started.md for 2.0

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update get-started.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/33
- Other reference link(s):

### CODE REVIEW

File: en/get-started.md [hunk 1]
```diff
@@ -34,15 +34,15 @@ For additional information about DM, please consult [Data Migration Overview](ov
 
 ## Setup
 
-We're going to deploy 3 instances of MySQL Server, and 1 instance each of pd-server, tikv-server, and tidb-server. Then we'll start a single DM-master and 3 instances of DM-worker.
+We're going to deploy 3 instances of MySQL Server, and 1 instance each of pd-server, tikv-server, and tidb-server. Then we'll start 3 instances of DM-master and 3 instances of DM-worker.
 
-1. Install MySQL 5.7, download and extract the TiDB v3.0 and DM v1.0.4 packages we'll use:
+1. Install MySQL 5.7, download and extract the TiDB v4.0 and DM v2.0 packages we'll use:
```
Comment (ran-huang) [inline en/get-started.md:39]: I think they are two separate sentences? We should use a period or semicolon to denote that.
```suggestion
1. Install MySQL 5.7. Then download and extract the TiDB v4.0 and DM v2.0 packages we'll use:
```

File: en/get-started.md [hunk 2]
```diff
@@ -34,15 +34,15 @@ For additional information about DM, please consult [Data Migration Overview](ov
...
```
Comment (ran-huang) [inline en/get-started.md:40]:
```suggestion

    {{< copyable "shell-regular" >}}

```

File: en/get-started.md [hunk 3]
```diff
@@ -179,61 +179,153 @@ Note that we have incrementing, non-overlapping IDs in the left-hand column. The
...
+    [10]+  Running                 dm-worker --config=dm-cnf/dm-worker"$i".toml --log-file=logs/dm-worker"$i".log >> logs/dm-worker"$i".log 2>&1 &
+    ```
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    ps -a
+    ```
+
+    ```
```
Comment (ran-huang) [inline en/get-started.md:231]:
```suggestion
    ```
       PID TTY          TIME CMD
```

File: en/get-started.md [hunk 4]
```diff
@@ -179,61 +179,153 @@ Note that we have incrementing, non-overlapping IDs in the left-hand column. The
...
+                "result": true,
+                "msg": "",
+                "source": "mysql-replica-03",
+                "worker": "worker3"
+            }
+        ]
+    }
+    ```
+
+Each of the upstream MySQL Server instances corresponds to a separate DM-worker instance, each of which has its own configuration file.
```
Comment (ran-huang) [inline en/get-started.md:294]:
```suggestion
Each of the upstream MySQL Server instances corresponds to a separate `source.toml`. Each DM-master and DM-worker instance has its own configuration file.
```

File: en/get-started.md [hunk 5]
```diff
@@ -179,61 +179,153 @@ Note that we have incrementing, non-overlapping IDs in the left-hand column. The
...
+
+name = "master1"
+master-addr = ":8261"
+advertise-addr = "127.0.0.1:8261"
+peer-urls = "127.0.0.1:8291"
+initial-cluster = "master1=http://127.0.0.1:8291,master2=http://127.0.0.1:8292,master3=http://127.0.0.1:8293"
 ```
 
-Each of the upstream MySQL Server instances corresponds to a separate DM-worker instance, each of which has its own configuration file. These files describe the details of the connection to the upstream MySQL Server as well as where to store the relay log files (the local copy of the upstream server's binary log) and the output of Mydumper. Each DM-worker should listen on a different port (defined by `worker-addr`). Here's dm-worker1.toml, for example:
+The following is an example of `dm-worker1.toml`:
```
Comment (ran-huang) [inline en/get-started.md:310]:
```suggestion
The following is an example of `dm-worker1.toml`:

{{< copyable "" >}}
```

File: en/get-started.md [hunk 6]
```diff
@@ -179,61 +179,153 @@ Note that we have incrementing, non-overlapping IDs in the left-hand column. The
...
 
 ```toml
 # Worker Configuration.
```
Comment (ran-huang) [inline en/get-started.md:313]:
```suggestion
# DM-worker1 Configuration.
```

File: en/get-started.md [hunk 7]
```diff
@@ -300,7 +393,7 @@ There are a number of global options, and several groups of options that define
 
 * We use `black-white-list` to limit the scope of this task to database `dmtest`.
 
-* The `loaders` section defines where to find the output of each instance of Mydumper that was executed by the respective instance of DM-worker.
+* The `loaders` section defines where to find the output of each instance of Mydumper that was executed by the respective MySQL source.
```
Comment (ran-huang) [inline en/get-started.md:396]:
```suggestion
* The `loaders` section defines where to find the output of each instance of Mydumper that has been executed by the respective MySQL source.
```

**Comment by toutdesuite** [comment]
@ran-huang PTAL again. Thanks~

File: en/get-started.md [hunk 8]
```diff
@@ -179,61 +181,156 @@ Note that we have incrementing, non-overlapping IDs in the left-hand column. The
 
 Our goal in this exercise is to use DM to combine the data from these distinct MySQL instances into a single table in TiDB.
 
-The package of configuration files we unpacked earlier (dm-cnf.tgz) contains the configuration for the components of the TiDB cluster, the DM components, and for the 2 DM tasks we'll explore in this tutorial.
+The package of configuration files we unpacked earlier (dm-cnf.tgz) contains the configuration for the components of the TiDB cluster, the DM components, and for a DM task we'll explore in this tutorial.
```
Comment (ran-huang) [inline en/get-started.md:184]:
```suggestion
The package of configuration files we have unpacked earlier (dm-cnf.tgz) contains the configuration for the components of the TiDB cluster, the DM components, and for a DM task we'll explore in this tutorial.
```

**Comment by toutdesuite** [comment]
@ran-huang Comments addressed. PTAL~

**Pull request merged**